examples/nlp/semantic_similarity_with_bert.py

```diff
@@ -0,0 +1,363 @@
+"""
+Title: Semantic Similarity with BERT
+Author: [Mohamad Merchant](https://twitter.com/mohmadmerchant1)
+Date created: 2020/08/15
+Last modified: 2020/08/20
+Description: Natural Language Inference by Fine-tuning BERT model on SNLI Corpus.
+"""
+"""
+## Introduction
+
+Semantic Similarity or Natural Languauge Inference is the task of determining how two
+sentences are similar to each other in terms of their meaning.
+This example demonstrates the use of SNLI (Standford Natural Language Inference) Corpus
+to perform semantic similarity with Transformers.
+We will fine-tune BERT model by providing two sentences as inputs and it outputs the
+probability of the similarity between sentences.
+
+### References
+* [BERT](https://arxiv.org/pdf/1810.04805.pdf)
+* [SNLI](https://nlp.stanford.edu/projects/snli/)
+"""
+
+"""
+## Setup
+"""
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+import transformers
+from transformers import BertTokenizer, TFBertModel
```

Comment:
You don't need this import line, you can just access these objects from `transformers`.